libraries/libmesosphere/source/kern_k_memory_block_manager.cpp

```
@@ -302,34 +293,30 @@ namespace ams::kern {
         while (remaining_pages > 0) {
             const size_t remaining_size = remaining_pages * PageSize;
-            KMemoryInfo cur_info = it->GetMemoryInfo();
 
             /* If we need to, create a new block before and insert it. */
-            if (cur_info.m_address != GetInteger(cur_address)) {
+            if (it->GetAddress() != cur_address) {
                 KMemoryBlock *new_block = allocator->Allocate();
 
                 it->Split(new_block, cur_address);
                 it = m_memory_block_tree.insert(*new_block);
                 it++;
```

cppcheck

[performance] Prefer prefix ++/-- operators for non-primitive types.

(postfixOperator)